cubequery/tasks/__init__.py

```diff
@@ -162,7 +168,7 @@ def validate_arg(self, name, value):
 
         return True, ""
 
-    def calculate_result(self, **kwargs):
+    def calculate_result(self, publish_to_esri, **kwargs):

```

Comment:
the publish functionality is deliberately not only for esri. can we change this to just publish please.